src/site/content/en/blog/cache-api-quick-guide/index.md

```diff
@@ -0,0 +1,350 @@
+const stringResponse = new Response('Hello world');
+```
+
+You can set the MIME type of a `Response` by setting the appropriate header.
+
+```js
+  const options = {
+    headers: {
+      'Content-Type': 'application/json'
+    }
+  }
+  const jsonResponse = new Response('{}', options);
+```
+
+If you have retrieved a `Response` and wish to access its body, there are
+several helper methods you can use. Each returns a `Promise` that resolves
+with a value of a different type.
+
+<table>
+  <thead>
+    <th>Method</th>
+    <th>Description</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>arrayBuffer</code></td>
+      <td>
+        Returns an <code>ArrayBuffer</code> containing the body, serialized to
+        bytes.
+      </td>
+    </tr>
+    <tr>
+      <td><code>blob</code></td>
+      <td>
+        Returns a <code>Blob</code>. If the <code>Response</code> was created
+        with a <code>Blob</code> then this new <code>Blob</code> has the same
+        type. Otherwise, the <code>Content-Type</code> of the
+        <code>Response</code> is used.
+      </td>
+    </tr>
+    <tr>
+      <td><code>text</code></td>
+      <td>
+        Interprets the bytes of the body as a UTF-8 encoded string.
+      </td>
+    </tr>
+    <tr>
+      <td><code>json</code></td>
+      <td>
+        Interprets the bytes of the body as a UTF-8 encoded string, then tries
+        to parse it as JSON. Returns the resulting object, or throws a
+        <code>TypeError</code> if the string cannot be parsed as JSON.
+      </td>
+    </tr>
+    <tr>
+      <td><code>formData</code></td>
+      <td>
+        Interprets the bytes of the body as an HTML form, encoded either as
+        <code>multipart/form-data</code> or
+        <code>application/x-www-form-urlencoded</code>. Returns a
+        <a href="https://developer.mozilla.org/en-US/docs/Web/API/FormData">FormData</a>
+        object, or throws a <code>TypeError</code> if the data cannot be parsed.
+      </td>
+    </tr>
+    <tr>
+      <td><code>body</code></td>
+      <td>
+        Returns a <a href="https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream">ReadableStream</a>
+        for the body data.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+For example
+
+```js
+const response = new Response('Hello world');
+const buffer = await response.arrayBuffer();
+console.log(new Uint8Array(buffer));
+// Uint8Array(11) [72, 101, 108, 108, 111, 32, 119, 111, 114, 108, 100]
+```
+
+## Retrieving from a cache {: #retreiving }
+
+To find an item in a cache, you can use the `match` method.
+
+```js
+const response = await cache.match(request);
+console.log(request, response);
+```
+
+If `request` is a string it is first be converted to a `Request` by calling
+`new Request(request)`. The function returns a `Promise` that resolves to
+a `Response` if a matching entry is found, or `undefined` otherwise.
+
+To determine if two `Requests` match, more than just the URL is used. Two
```

Comment:
```suggestion
To determine if two `Requests` match, the browser uses more than just the URL. Two
```
I think we're talking about the browser's logic but it's not clear to me because we don't specifically mention the subject of the sentence